src/apps/patron-page/sections/BasicDetailsSection.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { FC } from "react";
+import { PatronV5 } from "../../../core/fbs/model";
+import { useText } from "../../../core/utils/text";
+
+interface BasicDetailsSectionProps {
+  patron: PatronV5;
+}
+
+const BasicDetailsSection: FC<BasicDetailsSectionProps> = ({ patron }) => {
+  const t = useText();
+
+  return (
+    <section>
+      <h2 className="text-body-small-regular mt-32 mb-16">
+        {t("patronPageBasicDetailsHeaderText")}
+      </h2>
+      <div className="dpl-patron-info">
+        <div className="dpl-patron-info__label">
+          {t("patronPageBasicDetailsNameLabelText")}
+        </div>
+        <div className="dpl-patron-info__text">{patron.name}</div>
+        <div className="dpl-patron-info__label">
+          {t("patronPageBasicDetailsAddressLabelText")}
+        </div>
+        <div className="dpl-patron-info__text">
+          <div>{patron.address?.coName}</div>
```

Comment:
I think you should destruct the address earlier because all the properties are mandatory on the AddressV2 type. Then you dont have to check all of the individually.
Another problem is that the address is possibly undefined. What happens if it is undefined?

Reply:
fixed